air-script/tests/helpers.rs

```diff
@@ -8,43 +9,54 @@ pub enum TestError {
     IO(String),
     Parse(String),
     IR(String),
+    Gce(String),
 }
 
 pub struct Test {
-    input_path: String,
+    ir: AirIR,
 }
 
 impl Test {
-    pub fn new(input_path: String) -> Self {
-        Test { input_path }
-    }
-
-    pub fn transpile(&self) -> Result<String, TestError> {
+    pub fn new(input_path: String) -> Result<Self, TestError> {
```

Comment:
I'd prefer to keep these separate. The idea is that if they're separate then in the future we could offer different options - full transpilation via `transpile`, or only testing the `parsing` or `ir` generation steps